docs/src/add_to_browser.md

```diff
@@ -0,0 +1,63 @@
+# Adding Stract to the Web Browser 
+
+While it is not currently recommended to use Stract as your primary search option, you can add stract 
+to your browser by doing following the instructions listed under the following browsers. 
```

Comment:
I think either 'doing' or 'following' should be removed here.
how do you feel about the sentence
`[...] to your browser by following the instructions listed under your browser of choice here.`?

Reply:
Good catch! I should have done a final editing pass before making the merge request. Fixed.